# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/mesa/hotfix-nir.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- mesa-22.1.5/src/compiler/nir/nir_opt_move.c.vanilla	2022-08-08 23:47:13.093836454 +0200
+++ mesa-22.1.5/src/compiler/nir/nir_opt_move.c	2022-08-08 23:47:35.659835554 +0200
@@ -51,7 +51,7 @@
  * lower register pressure.
  */
 
-static ALWAYS_INLINE bool
+static bool
 src_is_ssa(nir_src *src, void *state)
 {
    return src->is_ssa;
